SparkPost/support-docs
# WE-296 Parse embedded html

**What Changed**
- Adds `rehype-raw` to parse embedded html

**How to verify**
- Visit the deploy preview or run the site locally
- Visit `/momentum/4/loam`
- Verify the embedded `<a>` tags from `loam.md` are parsed correctly
